EarlGrey/Synchronization/GREYRunLoopSpinner.m

```diff
@@ -294,15 +291,16 @@ - (CFRunLoopObserverRef)grey_setupObserverInMode:(CFStringRef)mode
  *
  *  @return The registered timer or @c nil if no timer was added to @c mode.
  */
-- (CFRunLoopTimerRef)grey_setupWakeUpTimerInMode:(CFStringRef)mode {
+- (CFRunLoopTimerRef)grey_setupWakeUpTimerInMode:(NSString *)mode {
   if (self.maxSleepInterval > 0) {
```

Comment:
It's an ivar, so I went ahead and changed it to be accessed directly and made the same change for other ivars in this file.